doc/association-rules.rst

```diff
@@ -0,0 +1,79 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+#### Inputs
+
+-  **Data**
+
+Data set
```

Comment:
If you align these description lines with four spaces under the list item heading, does it become part of the item? Currently, it's a separate paragraph that break the list.